### ISSUE TITLE
ssh: connect to host git.mydomain.ltd port 10022: No route to host

### ISSUE THREAD

**Issue by zedtux** [sameersbn/docker-gitlab#491]
I'm using the latest version of docker-gitlab (installed yesterday evening) and while everything else than pulling/pushing through SSH is working, when I update my git remote `origin` with that Gitlab suggest me (in this case `ssh://git@git.mydomain.ltd:10022/zedtux/my-repo.git`), I have the following:

```
~ • g pull
ssh: connect to host git.mydomain.ltd port 10022: No route to host
fatal: Could not read from remote repository.

Please make sure you have the correct access rights
and the repository exists.
```

Looking at my server, I have discovered that Docker is listening on port 10022 but on the IPv6 not IPv4:

```
$ netstat -putan
(Not all processes could be identified, non-owned process info
 will not be shown, you would have to be root to see it all.)
Active Internet connections (servers and established)
Proto Recv-Q Send-Q Local Address           Foreign Address         State       PID/Program name
tcp        0      0 0.0.0.0:22              0.0.0.0:*               LISTEN      -               
tcp        0      0 0.0.0.0:25              0.0.0.0:*               LISTEN      -               
tcp        0      0 127.0.0.1:6379          0.0.0.0:*               LISTEN      -               
tcp6       0      0 :::22                   :::*                    LISTEN      -               
tcp6       0      0 :::25                   :::*                    LISTEN      -               
tcp6       0      0 :::443                  :::*                    LISTEN      -               
tcp6       0      0 :::10080                :::*                    LISTEN      -               
tcp6       0      0 :::10022                :::*                    LISTEN      -               
tcp6       0      0 :::80                   :::*                    LISTEN      -               
```

How to force Docker/Gitlab to use the IPv4 ?

**Comment by zedtux** [comment]
For your information I have fixed my issue by updating my server's `/etc/sysctl.conf` file by uncommenting the `net.ipv6.conf.all.forwarding=1` and adding `net.ipv6.bindv6only=0`. Finally I have stopped all my containers, restarted docker and restarted the contains and now it's working.

There is a strange behavior in `netstat` as [described in this SO answer](http://stackoverflow.com/a/30006034).

I think, a documentation regarding this issue should be added to the `README.md` file so that people aren't going to waste time on investigating this issue.
